build.cmd

```diff
@@ -25,12 +24,12 @@ if /I "%1" == "/skiptests" set OptTest=$false&&shift&& goto :ParseArguments
 if /I "%1" == "/restore-only" set OptBuild=$false&&set OptDeploy=$false&&set OptTest=$false&&shift&& goto :ParseArguments
 if /I "%1" == "/no-deploy-extension" set OptDeploy=$false&&shift&& goto :ParseArguments
 if /I "%1" == "/diagnostic" set OptLog=$true&&shift&& goto :ParseArguments
-if /I "%1" == "/integrationtests" set OptDeployDeps=$true&&set OptIntegrationTest=$true&&shift&& goto :ParseArguments
+if /I "%1" == "/integrationtests" OptIntegrationTest=$true&&shift&& goto :ParseArguments
```

Comment:
Looks like you lost a `set` here.

Reply:
`OptDeployDeps` is not used anywhere in the codebase with this change.

Reply:
Sure, but you don't have the `set` before `OptIntegrationTest`, so runing `build.cmd /integrationtests` errors out.

Reply:
ah, sorry. fixed